pkg/specgenutil/specgen.go

```diff
@@ -491,6 +491,12 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *entities.ContainerCreateOptions
 		s.Labels = labels
 	}
 
+	// Intel RDT CAT
+	s.IntelRdt = &specs.LinuxIntelRdt{}
```

Comment:
Is there a reason to set this, instead of leaving it defaulted if the user did not provide one?

Reply:
good hint, I've adapted it.